src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -970,6 +973,11 @@ def customizePerJob(self, job):
         if job.get('inputDatasetLocations'):
             jdl.append('+DESIRED_CMSDataLocations = "%s"\n' % ','.join(job['inputDatasetLocations']))
 
+        # HighIO jobs
+        highIOjob = job.get('highIOjob')
+        if highIOjob:
+            jdl.append('+Requestioslots = 1\n')
```

Comment:
Ah,  your right. Those types has to have one location. I am just wondering whether it still can get submitted site without highIO sites (if that one site doesn't have highIO slot).